### PR TITLE
Js

### PR DESCRIPTION
**Pull request by nata13120403**

### CODE REVIEW

**Comment by nata13120403** [comment]
Are you still hiding?

File: JS/index1.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+  <div id="divDate">  </div>
+  <script>
+    var div =  document.getElementById("divDate");
+    div.innerHTML = new Date().toDateString;
+  </script>
+  <script>
+      alert ('Hello, word!');
+  </script>
```
Comment (anton-shevchook) [inline JS/index1.html:18]: You don't need to create multiple <script></script> tags.
You can write your whole code in one code script.

Comment (nata13120403) [reply JS/index1.html:18]: Thanks!

**Review by anton-shevchook — APPROVED** [review]
Good job!

**Pull request merged**